.github/workflows/tests.yml

```
@@ -0,0 +1,34 @@
+    strategy:
+      matrix:
+        test-suite: ['phpcsfixer', 'phpstan', 'rector', 'phpcs']
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7.0.1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@f3e473d116dcccaddc5834248c87452386958240 # 2.37.2
+        with:
+          php-version: ${{ matrix.php }}
```

actionlint (1.7.12)

[error] property "php" is not defined in object type {test-suite: string}

(expression)